src/ert/scheduler/job.py

```diff
@@ -186,12 +186,24 @@ async def _handle_failure(self) -> None:
         logger.error(error_msg)
         log_info_from_exit_file(Path(self.real.run_arg.runpath) / ERROR_file)
 
+    async def _handle_aborted(self) -> None:
+        info_msg = "Scheduler cancelled, stopping job"
```

Comment:
Is this a scenario that did not exist in the job_queue code? (since we are introducing a brand new type of error message that the users can see)

Reply:
Updated accordingly.